packages/lodestar/src/util/expandYamlValue.ts

```diff
@@ -0,0 +1,32 @@
+import {AnySSZType, FullSSZType, Type, parseType} from "@chainsafe/ssz";
+import {BitList, BitVector} from "@chainsafe/bit-utils";
+
+export function expandYamlValue(value: any, type: AnySSZType): any {
```

Comment:
this should probably go in another package at some point, we will need this for the ssz cli

Reply:
also this is copy-pastad from lodestar/test/utils, so that old one should go away at some point. just documenting it here

Reply:
I forgot to delete that, I've actually edited #340 to include it